lib/handlers/quote/quote.ts

```diff
@@ -57,6 +58,13 @@ export class QuoteHandler extends APIGLambdaHandler<
     let result: Response<QuoteResponse> | ErrorResponse
 
     try {
+      for (const chainId of SUPPORTED_CHAINS) {
+        if (GlobalRpcProviders.getGlobalUniRpcProviders(log).has(chainId)) {
+          const provider = GlobalRpcProviders.getGlobalUniRpcProviders(log).get(chainId)!
+          provider.forceAttachToNewSession()
+        }
+      }
+
```

Comment:
I think we should be able to use the `chainId` defined in line 47, since a given request will only execute for a single chain.